src/components/ProcessCreate/CreationProgress.tsx

```diff
@@ -43,24 +43,28 @@ export const CreationProgress = ({ error, sending, step }: CreationProgressProps
   }, [step])
 
   return (
-    <Stack direction='column' gap={4}>
-      <Text>{t('process_create.creation_steps_description')}</Text>
-      <Heading as='h2' size='md' textAlign='center'>
-        {t('process_create.creation_steps_title')}
-      </Heading>
+    <Stack direction='column' gap={5} p={0}>
+      <Box bgImage='url(/assets/spreadsheet-confirm-modal.png)' height='150px' bgSize='cover' borderRadius='lg' />
+      <Text mb={6} textAlign='center'>
+        {t('process_create.creation_steps_description')}
+      </Text>
       <List spacing={3}>
-        {Object.keys(labels).map((key) => (
-          <ListItem key={key}>
-            <ListIcon
-              as={steps[key as keyof CreationStepsState] ? AiFillCheckCircle : AiFillCloseCircle}
-              fontSize={22}
-            />
-            {labels[key]}
+        {Object.keys(labels).map((key, index) => (
+          <ListItem key={key} display='flex' alignItems='center' gap={2}>
+            {steps[key as keyof CreationStepsState] ? (
+              <ListIcon as={AiFillCheckCircle} fontSize={23} m={0} />
+            ) : (
+              <>{!error ? <Spinner boxSize={5} mr='3px' /> : <ListIcon as={AiFillCloseCircle} fontSize={23} m={0} />}</>
```

Comment:
Were you able to try out if can we use the `<Spinner />` component inside an Icon one? (in this case, inside a `ListIcon` one).

Reply:
It is not possible to use the Spinner component as an Icon